### ISSUE TITLE
Plot training loss against validation loss to check for overfitting.

### ISSUE THREAD

**Issue by sydneyvert** [dj311/uob-summer-project#43]

**Comment by sydneyvert** [comment]
https://machinelearningmastery.com/learning-curves-for-diagnosing-machine-learning-model-performance/

**Comment by samanthawise** [comment]
[Display Deep Learning Model Training History in Keras](https://machinelearningmastery.com/display-deep-learning-model-training-history-in-keras/)

[Model visualization](https://keras.io/visualization/)

**Comment by dj311** [comment]
https://github.com/dj311/uob-summer-project/commit/dd31cbc487d6d080d57347924efbf11dd0350a38
